app-misc/graphlcd-base/graphlcd-base-1.0.2.ebuild

```diff
@@ -0,0 +1,87 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit flag-o-matic udev
+
+DESCRIPTION="Contains the lowlevel lcd drivers for GraphLCD"
+HOMEPAGE="https://projects.vdr-developer.org/projects/graphlcd-base"
+SRC_URI="https://projects.vdr-developer.org/git/${PN}.git/snapshot/${P}.tar.bz2"
+
+KEYWORDS="~amd64 ~ppc ~x86"
+SLOT="0"
+LICENSE="GPL-2"
+IUSE="fontconfig freetype g15 graphicsmagick imagemagick lcd_devices_ax206dpf lcd_devices_picolcd_256x64 lcd_devices_vnc"
+REQUIRED_USE="?? ( graphicsmagick imagemagick )"
+
+RDEPEND="
+	dev-libs/libhid
+	net-libs/libvncserver
+	freetype? ( media-libs/freetype:2= )
+	fontconfig? ( media-libs/fontconfig:1.0= )
+	g15? ( app-misc/g15daemon )
```

Comment:
This IUSE doesn't seem to be in any kind of use in this ebuild. Move it to optfeature or depend unconditionally, since introducing runtime IUSE is against gentoo policy?

Reply:
Good catch, thanks.